### DISCUSSION TITLE
What do you call a 10-sided polygon?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#860] [Galaxy Trivia] [answered]
**Bonus:** What is interior angle sum?

**Comment by ppnwt** [comment]
The sum of all the interior angles of the 10-sided is 1,440

**Comment by arpittyagi102** [comment]
A decagon

**Comment by Carol42** [comment, marked as answer]
In geometry, a **decagon** is a ten-sided polygon or 10-gon. The total sum of the interior angles of a simple decagon is 1440°.

![image](https://github.com/ParanoidUser/yolo/assets/63017741/b45668f7-810d-48d9-b1ff-2620106d20e4)

**Comment by peterbaccaro** [comment]
A 10-sided polygon is called a **decagon**.

The interior angle sum of a polygon can be calculated using the formula: (n - 2) × 180 degrees, where "n" represents the number of sides in the polygon.

For example, for a decagon (a polygon with 10 sides), the interior angle sum can be found by substituting "n" with 10:

(10 - 2) × 180 = 8 × 180 = 1440 degrees

Therefore, the interior angle sum of a decagon is **1440 degrees**.
